docs/examples/open5gs/eupf-bgp.yaml

```diff
@@ -0,0 +1,157 @@
+---
+
+args:
+  - --config
+  - /app/conf/config.yml
+
+securityContext:
+  privileged: true
+
+podSecurityContext:
+  sysctls:
+  - name: net.ipv4.ip_forward
+    value: "1"
+
+deploymentStrategy:
+  type: Recreate
+
+configMaps:
+  config:
+    data:
+      config.yml: |
+        interface_name: eth0
+        api_address: :8080
+        pfcp_address: :8805
+        metrics_address: :9090
+  bird:
+    data:
+      bird.conf: |
+        router id 10.233.64.1;
+
+        protocol bfd {
+        }
+
+        protocol direct {
+          interface "*";
+        }
+
+        protocol kernel {
+          learn;
+          scan time 5;
+          import all;
+          export all;
+        }
+
+        protocol device {
+          scan time 5;
+        }
+
+        protocol static eupf {
+          route 10.45.0.0/16 reject;
```

Comment:
Is it possible to have this subnet as a common parameter with smf and other components?

Reply:
with existing tools - it's complicated :(

Reply:
Will discuss it later